examples/shader/custom_shader_manual.rs

```diff
@@ -0,0 +1,143 @@
+use bevy::{
+    core_pipeline::Transparent3d,
+    pbr::{
+        DrawMesh, MeshPipeline, MeshPipelineKey, MeshUniform, SetMeshBindGroup,
+        SetMeshViewBindGroup,
+    },
+    prelude::*,
+    render::{
+        render_phase::{AddRenderCommand, DrawFunctions, RenderPhase, SetItemPipeline},
+        render_resource::*,
+        view::{ComputedVisibility, ExtractedView, Msaa, Visibility},
+        RenderApp, RenderStage,
+    },
+};
+
+fn main() {
```

Comment:
This example has stopped rendering things. I also don't think we should use "material" terminology here anymore because Material is now a "first class" type in Bevy.

Reply:
Maybe we should just remove this example for now so we can merge this pr in time for the release?

Reply:
I removed the example.